cpp/src/io/parquet/page_data.cu

```diff
@@ -449,8 +449,9 @@ __global__ void __launch_bounds__(decode_block_size)
   int out_thread0;
   [[maybe_unused]] null_count_back_copier _{s, t};
 
+  auto const mask = decode_kernel_mask::GENERAL;
   if (!setupLocalPageInfo(
-        s, &pages[page_idx], chunks, min_row, num_rows, mask_filter{KERNEL_MASK_GENERAL}, true)) {
+        s, &pages[page_idx], chunks, min_row, num_rows, mask_filter{mask}, true)) {
```

Comment:
I have a mild preference for just 
`mask_filter{decode_kernel_mask::GENERAL}`
so your eyes don't have to go hunting for where mask is declared.

Reply:
I was trying to keep all the args on one line 😭 I'll change it.